File: blog_springBoot/src/main/java/com/synergism/blog/util/base64Util.java
package com.synergism.blog.util;


import java.util.Base64;

public class base64Util {

    public static String encode(String src){
        return new String(Base64.getEncoder().encode(src.getBytes()));
    }

    public static String decode(String src){
        return new String(Base64.getDecoder().decode(src.getBytes()));
    }

}

File: blog_vue/src/utils/Base64Util.ts
export default class Base64Util {
  public static encode(str: string): string {
    // eslint-disable-next-line @typescript-eslint/no-var-requires
    const Base64 = require("js-base64").Base64;
    return Base64.encode(str);
  }

  public static decode(str: string): string {
    // eslint-disable-next-line @typescript-eslint/no-var-requires
    const Base64 = require("js-base64").Base64;
    return Base64.decode(str);
  }
}
